scripts/1.download.py

```diff
@@ -0,0 +1,52 @@
+
+# coding: utf-8
```

Comment:
You could make multiple files starting with two, if you wanted to show they were parallel. Not sure whether that will turn out to be a good practice or not